modules/runtime-info-ui/src/main/resources/ui/index.html

```diff
@@ -5,104 +5,8 @@
-                        var value = field;
-                        if (typeof field !== "undefined" && field !== null) {
-                            if (field instanceof Array) {
-                                value = field.toString();
-                            } else if (field instanceof Date) {
-                                value = field.toLocaleDateString() + ", " + field.toLocaleTimeString();
-                            }
-                            $container.html(value);
-                            $container.parent().show();
-                        } else {
-                            $container.parent().hide();
-                        }
-                    },
-                    dbConsistent = false,
-                    bundlesConsistent = false;
-
-            $.getJSON("/sysinfo/bundles/version", function (data) {
-                var $message = $sysInfoContainer.find("div#build span.message"),
-                        $messageDiv = $sysInfoContainer.find("div#build"),
-                        $version = $sysInfoContainer.find("div#version span"),
-                        $buildNumber = $sysInfoContainer.find("div#build-number span"),
-                        versions = [],
-                        buildNumbers = [];
-
-
-                if (data.consistent) {
-                    $message.html("Opencast is running with consistent bundles version.");
-                    $messageDiv.addClass("ok");
-
-                    checkAndDisplay(data.version, $version);
-                    checkAndDisplay(data.buildNumber, $buildNumber);
-                } else {
-                    $message.html("Opencast is running with inconsistent bundles version. Update the deprecated bundles to avoid loss or corruption of data.");
-                    $buildNumber.addClass("error");
-
-                    if (typeof data.versions !== "undefined") {
-                        $.each(data.versions, function (index, element) {
-                            if ($.inArray(element.version, versions)) {
-                                versions.push(element.version);
-                            }
-
-                            if ($.inArray(element.buildNumber, buildNumbers)) {
-                                buildNumbers.push(element.buildNumber);
-                            }
-                        });
-                    }
-
-                    if (versions.length > 0) {
-                        $version.before("Version" + (versions.length > 1 ? "s: " : ": "));
-                        checkAndDisplay(versions, $version);
-                    }
-
-                    if (buildNumbers.length > 0) {
-                        $buildNumber.before("Build number" + (buildNumbers.length > 1 ? "s: " : ": "));
-                        checkAndDisplay(buildNumbers, $buildNumber);
-                    }
-                }
-            });
-        });
-    </script>
+    <script type="text/javascript" src="js/jquery/dist/jquery.min.js"></script>
```

Comment:
No strong opinion, but do we want the "dist" in the URL? "js/jquery.min.js" would be nicer ;-)

What do you think?

Reply:
We want. That makes the deployment much easier in case there are multiple libraries at one point. And really no one will care about this path ;-P